src/client/lobby/room-list.js

```diff
@@ -42,11 +42,11 @@ export class RoomList extends Component {
                  <div className="col-lg-12 text-center">
                     <div className="col-lg-12 list-group">
                         <div className="row">
-                            <button className={`fa fa-cog fa-3x btn btn-info ${lobbyStyle['cogs-button']}`}
+                            <button className={`fa fa-cog fa-3x btn ${lobbyStyle['cogs-button']}`}
```

Comment:
Let's make a build change so we can reference this as lobbyStyle.cogsButton (no change to the scss file)